app/assets/stylesheets/application.css.scss

```diff
@@ -21,46 +21,19 @@
 
 /*  @import url(http://fonts.googleapis.com/css?family=Roboto:100,300,600);
 */
-#flat-slider.ui-slider {
-background: #d5cebc;
-border: none;
-border-radius: 0; }
 
-#flat-slider.ui-slider .ui-slider-handle {
-width: 20px;
-height: 20px;
-border-radius: 50% 50% 0;
-border-color: transparent;
-transition: border 0.4s ease; }
-
-#flat-slider.ui-slider .ui-slider-handle.ui-state-hover,
-#flat-slider.ui-slider .ui-slider-handle.ui-state-focus,
-#flat-slider.ui-slider .ui-slider-handle.ui-state-active {
-border-color: #172f38; }
-
-#flat-slider.ui-slider .ui-slider-pip .ui-slider-line {
-background: #d5cebc;
-transition: all 0.4s ease; }
-
-#flat-slider.ui-slider.ui-slider-horizontal {
-height: 6px; }
-
-#flat-slider.ui-slider.ui-slider-horizontal .ui-slider-handle {
--webkit-transform: rotateZ(45deg);
-        transform: rotateZ(45deg);
-top: -25px;
-margin-left: -10px; }
-
-#flat-slider.ui-slider.ui-slider-horizontal .ui-slider-pip {
-top: 10px; }
-
-#flat-slider.ui-slider.ui-slider-horizontal .ui-slider-pip .ui-slider-line {
-  width: 2px;
-  height: 10px;
-  margin-left: -1px; }
-
-#flat-slider.ui-slider.ui-slider-horizontal .ui-slider-pip[class*=ui-slider-pip-selected] .ui-slider-line {
-  height: 20px; }
-
-#flat-slider.ui-slider.ui-slider-horizontal .ui-slider-pip.ui-slider-pip-inrange .ui-slider-line {
-  height: 12px; }
+body {
+  background: image-url("background.jpg") no-repeat center center fixed;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+}
+
+h1 {
+  color: #ffffff;
+}
+
+h3 {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.